File: Workshop 2/The Happy Pirate/src/Users/Member.java
package Users;

import Model.Boat;

import javax.xml.bind.annotation.XmlElement;
import java.util.ArrayList;

/**
 * Created by henry on 2016-09-20.
 */
public class Member {
    //private variables
    private ArrayList<Boat> boats = new ArrayList<Boat>();
    private String firstname,
            lastname,
            personnumber,
            id,
            email;



    //constructor, getters and setters
    public Member(XmlElement data){

    }
    public String getId(){ return id; }


    // printing out info
    public String verboseInfo(){
        return "";
    }
    public String compactInfo(){
        return "";
    }
}

File: Workshop 2/The Happy Pirate C#/YatchClub/YatchClub/Users/Member.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml;
using YatchClub.Model;

namespace YatchClub.Users
{
    class Member
    {
        //private variables
        private List<Boat> boats = new List<Boat>();
        private string firstname,
                lastname,
                personnumber,
                id,
                email;



        //constructor, getters and setters
        public Member(XmlNode data)
        {

        }
        public string getId() { return id; }


        // printing out info
        public string verboseInfo()
        {
            return "";
        }
        public string compactInfo()
        {
            return "";
        }
    }
}
